src/javascript_bindings.pyx

```diff
@@ -29,14 +29,15 @@ cdef class JavascriptBindings:
     cpdef py_void SetFunction(self, py_string name, object func):
         self.SetProperty(name, func)
 
-    cpdef py_void SetObject(self, py_string name, object obj):
+    cpdef py_void SetObject(self, py_string name, object obj,
+                            py_bool allow_properties=False):
```

Comment:
Just check if the requested method is actually a method and that is sufficient, with code like this:
`if methodName in self.objects[objectName] and callable(self.objects[objectName]):`. This is required for V8 handler so that it throws an appropriate js exception in case method does not exist.